src/include/misc.h

```diff
@@ -483,5 +483,17 @@ union __wt_rand_state {
             WT_ERR(__wt_buf_extend(session, buf, (buf)->size + __len + 1));     \
         }                                                                       \
     } while (0)
-
+/*
+ * __wt_atomic_decrement_if_positive --
+ *     Use compare and swap to atomically decrement value by 1 if it's positive.
+ */
+static void inline __wt_atomic_decrement_if_positive(uint32_t *valuep)
```

Comment:
The typical way to declare these, if you look at `src/include/*_inline.h` is with `WT_INLINE` as that has correct attributes for all situations and systems
```
static WT_INLINE void
__wt_atomic...
```